implementations/ApiCatalog.json

```diff
@@ -1,38 +1,95 @@
 {
-  "name": "apicatalog.com",
-  "implementation": "Iron Verifiable Credentials",
-  "verifiers": [{
-    "id": "https://vc.apicatalog.com",
-    "endpoint": "https://vc.apicatalog.com/credentials/verify",
-    "supportedEcdsaKeyTypes": ["P-256", "P-384"],
-    "tags": ["Ed25519Signature2020", "vc2.0", "eddsa-rdfc-2022", "ecdsa-rdfc-2019"]
-  }],
-  "issuers": [{
-    "id": "did:key:z6Mkska8oQD7QQQWxqa7L5ai4mH98HfAdSwomPFYKuqNyE2y",
-    "endpoint": "https://vc.apicatalog.com/credentials/issue",
-    "options": {
-      "type": "Ed25519Signature2020"
-    },
-    "tags": ["Ed25519Signature2020", "vc2.0"]
-  }, {
-    "id": "did:key:z6Mkska8oQD7QQQWxqa7L5ai4mH98HfAdSwomPFYKuqNyE2y",
-    "endpoint": "https://vc.apicatalog.com/credentials/issue",
-    "options": {
-      "type": "eddsa-2022"
-    },
-    "tags": ["eddsa-rdfc-2022"]
-  }, {
-    "id": "did:key:zDnaepBuvsQ8cpsWrVKw8fbpGpvPeNSjVPTWoq6cRqaYzBKVP",
-    "endpoint": "https://vc.apicatalog.com/credentials/issue",
-    "options": {
-      "type": "ecdsa-2019"
-    },
-    "supportedEcdsaKeyTypes": ["P-256", "P-384"],
-    "tags": ["ecdsa-rdfc-2019"]
-  }],
-  "vpVerifiers": [{
-    "id": "https://vc.apicatalog.com",
-    "endpoint": "https://vc.apicatalog.com/presentations/verify",
-    "tags": ["Ed25519Signature2020", "vc2.0", "eddsa-rdfc-2022", "ecdsa-rdfc-2019"]
-  }]
+	"name": "apicatalog.com",
+	"implementation": "Iron Verifiable Credentials",
+	"verifiers": [
+		{
+			"id": "https://apicatalog.com/vc/verifier",
+			"endpoint": "https://vc.apicatalog.com/credentials/verify",
+			"supportedEcdsaKeyTypes": [
+				"P-256",
+				"P-384"
+			],
+			"tags": [
+				"Ed25519Signature2020",
+				"vc2.0",
+				"eddsa-rdfc-2022",
+				"ecdsa-rdfc-2019",
+				"ecdsa-sd-2023"
+			]
+		}
+	],
+	"issuers": [
+		{
+			"id": "https://apicatalog.com/issuer/Ed25519Signature2020",
+			"endpoint": "https://vc.apicatalog.com/credentials/issue",
+			"options": {
+				"type": "Ed25519Signature2020"
```

Comment:
we do still support this, but the latest VC-API has dropped `options.type`. each endpoint should handle a single signature type.

Reply:
@aljones15  A separate endpoint per issuers means much more work on my side, is there a way how to pass the information from test suite to an endpoint as an option/a parameter?

Reply:
We do still support `options.type` so that still works. Whatever you stick in options should be posted to the endpoint by the test. The newer pattern that works is to associate an issuer id with a keyType so you could have an endpoint where when it receives a vc with an issuer id of the form `did:key:zDna` and then uses an ecdsa p-256 key to sign and when the vc's issuer is `did:key:z2fa` it uses ecdsa `p-384` key. In the later case the endpoint id would need to be different for the `p-256` and for the `p-384` so the endpoint knows which issuer to use. One thing I like about the issuer method is that you know exactly which did key to use, the thing I don't like about it is that you have to know the did before posting and the did doesn't tell you exactly the format of the signer.

Reply:
> In the later case the endpoint id would need to be different for the p-256 and for the p-384 so the endpoint knows which issuer to use.

Issuers won't have different endpoints for these in production, will they? I fear sending people down the wrong implementation path by encouraging them to do things (like the query strings here) to accommodate the testing infrastructure.

Is this a more general issue? or a limitation in how we're configuring tests?